java/code/src/com/redhat/rhn/frontend/xmlrpc/errata/ErrataHandler.java

```diff
@@ -1106,6 +1111,8 @@ private Object getRequiredAttribute(Map map, String attribute) {
      *  String "advisory_type" the type of advisory for the errata (Must be one of the
      *          following: "Security Advisory", "Product Enhancement Advisory", or
      *          "Bug Fix Advisory"
+     *  String "advirory_status" the status of advisory for the errata
+     *         (Must be one of the following: "final", "testing", "retracted")
```

Comment:
'stable' is missing. Just for completness

Reply:
Aligned now.